oak_restricted_kernel/layout.ld

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
 */
 
-ENTRY(_start)
+ENTRY(_oak_start)
```

Comment:
Why was in changed form the standard `start` to `oak_start`?

Reply:
`_start` is also used by libc, and tests are linked against libc. Thus, for test builds, these two functions would clash.

I guess another option would be to wrap the `global_asm` stanza in a `#[cfg(not(test))]` to ensure this file wouldn't be compiled in at all; but I think this is slightly cleaner.